### PR TITLE
Feat/3582 change "shared space" and "workspace" to "space" in the UI

### PR DESCRIPTION
**Pull request by gbonaspetti**
## Checkpoints

#3582 

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)

**For code reviewers**

- [ ] The code is clear enough
- [ ] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [ ] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- [ ] Manual, quality tests have been done

### CODE REVIEW

**Review by raphj — APPROVED** [review]

File: frontend_app_admin_workspace_user/i18next.scanner/en/translation.json [hunk 1]
```diff
@@ -42,18 +42,14 @@
...
   "Error, invalid email address": "Error, invalid email address",
-  "Error while checking username availability": "Error while checking username availability",
   "Create a space": "Create a space",
   "Home": "Home",
   "If you do not link an email to this new user, please notify manually the username and password.": "If you do not link an email to this new user, please notify manually the username and password.",
   "Users": "Users",
   "Administration": "Administration",
   "Username": "Username",
-  "This username is not available": "This username is not available"
+  "Allowed characters: {{allowedCharactersUsername}}": "Allowed characters: {{allowedCharactersUsername}}"
```
Comment (raphj) [inline frontend_app_admin_workspace_user/i18next.scanner/en/translation.json:54]: Are changes on `frontend_app_admin_workspace_user/i18next.scanner/*/translation.json` files okay? They do not seem relevant to this PR.

File: frontend_app_folder_advanced/i18next.scanner/en/translation.json [hunk 2]
```diff
@@ -11,5 +11,6 @@
   "Restore": "Restore",
   "This folder is archived": "This folder is archived",
   "This folder is deleted": "This folder is deleted",
-  "Error while loading Tracim's content type list": "Error while loading Tracim's content type list"
+  "Error while loading Tracim's content type list": "Error while loading Tracim's content type list",
+  "The content has been modified by {{author}}": "The content has been modified by {{author}}"
```
Comment (raphj) [inline frontend_app_folder_advanced/i18next.scanner/en/translation.json:15]: What this a missing string?

File: frontend_app_custom-form/i18next.scanner/pt/translation.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "Last version": "Última versão",
+  "A content with the same name already exists": "Um conteúdo com o mesmo nome já existe",
+  "Error while creating document": "Erro ao criar o documento",
+  "New form": "Novo formulário",
+  "Validate and create": "Validar e criar",
+  "Document's title": "Título do documento",
+  "Cancel": "Cancelar",
+  "Validate": "Validar",
+  "Title undefined": "Título indeterminado"
+}
```
Comment (raphj) [inline frontend_app_custom-form/i18next.scanner/pt/translation.json:29]: Some bonus Portuguese translation? :-)

File: backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po [hunk 4]
```diff
@@ -94,11 +94,11 @@ msgstr ""
 
 #: tracim_backend/lib/core/workspace.py:400
 msgid "Workspace"
-msgstr "Shared space"
+msgstr "Space"
```
Comment (raphj) [inline backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po:97]: Should I send push a commit commit to fix the string in the python file?

 Otherwise, here are the changes to make in `tracim_backend/lib/core/workspace.py`:

```python
        query = self._base_query_without_roles().filter(
            Workspace.label.ilike("{0}%".format(_("Workspace")))
        )

        return _("Workspace {}").format(query.count() + 1)
```

Becomes:

```python
        query = self._base_query_without_roles().filter(
            Workspace.label.ilike("{0}%".format(_("Space")))
        )

        return _("Space {}").format(query.count() + 1)
```

File: backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po [hunk 5]
```diff
@@ -94,11 +94,11 @@ msgstr ""
 
 #: tracim_backend/lib/core/workspace.py:400
 msgid "Workspace"
```
Comment (raphj) [inline backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po:96]: With the modifications to the Python file:

```suggestion
#: tracim_backend/lib/core/workspace.py:419
msgid "Space"
```

File: backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po [hunk 6]
```diff
@@ -94,11 +94,11 @@ msgstr ""
...
-msgstr "Shared space"
+msgstr "Space"
 
 #: tracim_backend/lib/core/workspace.py:403
 msgid "Workspace {}"
```
Comment (raphj) [inline backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po:100]: With the modifications to the Python file:
```suggestion
#: tracim_backend/lib/core/workspace.py:422
msgid "Space {}"
```

**Review by raphj — CHANGES_REQUESTED** [review]

File: frontend_app_admin_workspace_user/i18next.scanner/en/translation.json (hunk 1 above)
Comment (gbonaspetti) [reply frontend_app_admin_workspace_user/i18next.scanner/en/translation.json:54]: These are automatic changes that came with the `build-translation` command, used to see if I didn't forget to update any key or translation.

File: backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po (hunk 4 above)
Comment (inkhey) [reply backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po:97]: I don't see any drawback to do so as soon as you correctly updated the translation scripts (for all lang because the key as changed !), but as we do currently use the term "workspace" everywhere in backend and we do not decide to change it, i don't see any advantage to do a very partial conversion now.

Do as you want, this seems to me not a really important point.

Comment (raphj) [reply backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po:97]: It seems like a string that is exposed to the user, so we should definitely update this I think.

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 7]
```diff
@@ -108,11 +108,11 @@ msgstr "Nouveau dossier {0}"
 
 #: tracim_backend/lib/core/workspace.py:400
 msgid "Workspace"
```
Comment (raphj) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po:110]:
```suggestion
msgid "Space"
```

File: backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po [hunk 8]
```diff
@@ -108,11 +108,11 @@ msgstr "Nouveau dossier {0}"
...
-msgstr "Espace partagé"
+msgstr "Espace"
 
 #: tracim_backend/lib/core/workspace.py:403
 msgid "Workspace {}"
```
Comment (raphj) [inline backend/tracim_backend/locale/fr/LC_MESSAGES/tracim_backend.po:114]:
```suggestion
msgid "Space {}"
```

File: backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po (hunk 4 above)
Comment (inkhey) [reply backend/tracim_backend/locale/en/LC_MESSAGES/tracim_backend.po:97]: You're right, this can cause issue for uncompleted new translation (english case is handled here).

**Review by raphj — APPROVED** [review]

**Comment by gbonaspetti** [comment]
This PR was closed because it was made with the wrong profile. The official PR is #3608 .
